### PR TITLE
Add notices any time we save the key 

### PR DESCRIPTION
**Pull request by borkweb**
Additionally, clear notices when saving the key.

Related: https://github.com/moderntribe/the-events-calendar/pull/987

See: https://central.tri.be/issues/65378

### CODE REVIEW

**Comment by bordoni** [comment]
🎱

File: src/Tribe/PUE/Checker.php [hunk 1]
```diff
@@ -568,6 +571,8 @@ public function validate_key( $key ) {
 					}
 				}
 
+				$pue_notices->clear_notices( $plugin_name );
```
Comment (barryhughes) [inline src/Tribe/PUE/Checker.php]: Just checking: are we positive we wish to clear notices at this point (during ajax validation, before a key has been committed/saved)?

Comment (borkweb) [inline src/Tribe/PUE/Checker.php]: When a valid license key is entered and gets ajax verified, it actually _does_ get saved. The only time the ajax verification doesn't get saved is when the key is invalid, requires an upgrade, or expired.

Comment (barryhughes) [inline src/Tribe/PUE/Checker.php]: You're totally right, I missed that completely :sleepy:

**Pull request merged**